autoload/lsp/util.vim

```diff
@@ -112,8 +112,10 @@ export def LspFileToUri(fname: string): string
   var uri: string = fname->fnamemodify(':p')
 
   if has("win32unix")
-    # We're in Cygwin
-    uri = CygwinToWindowsPath(uri)
+    # We're in Cygwin, we need to convert POSIX style paths to Windows style.
+    # `cygpath -m` converts paths of the form "/cygdrive/c/foo/bar" to "C:/foo/bar", and
```

Comment:
not sure the comment was accurate or not, since i recall not all possible it was /cygdrive/ header, it was not always was cygwin64/, but if just an example, then ok.

Reply:
> i have no Windows pc anymore, please try to self-verify a bit if all worked ok.

I did.

> not sure the comment was accurate or not, since i recall not all possible it was /cygdrive/ header, it was not always was cygwin64/, but if just an example, then ok.

I'm not sure I understand, can you please rephrase? Quick test in bash under cygwin:

```
$ cygpath -m /cygdrive/c/foo/bar
C:/foo/bar
$ cygpath -m /home/pete/foo/bar
C:/cygwin64/home/pete/foo/bar
```

If you mean that the path where cygwin is installed is not always `C:/cygwin64` then yes, that's true. I would hope in that case `cygpath` would return the actual correct path.

Reply:
if it is not sure, then perhaps can just remove those comments, 
since the command name supposed had already identify your code purpose.

Reply:
e.g just `convert POSIX style path to Windows style`